.circleci/config.yml

```diff
@@ -8,11 +8,11 @@ reference:
   android_config: &android_config
     working_directory: *workspace
     macos:
-      xcode: "11.5.0"
+      xcode: '11.5.0'
```

Comment:
Were these changes necessary?

Reply:
No just automatic formatting by my editor. I'll get rid of them